packages/eslint-plugin-ecmascript-compat/lib/features/es2016.spec.js

```diff
@@ -11,7 +11,16 @@ const ruleTester = new RuleTester({
 });
 
 ruleTester.run('compat', require('../rule'), {
-  valid: [],
+  valid: [
+    {
+      code: 'foo.includes();',
+      options: [{ polyfills: ['Array.prototype.includes'] }],
+    },
+    {
```

Comment:
For the purpose of what we're testing, I don't think it's necessary to test both the direct prototype usage here in addition to the prototype traversal usage above. One of the two is enough.

(applies to many files - I won't duplicate this comment